docs/content/development/extensions-contrib/graphite.md

```diff
@@ -52,10 +53,15 @@ The path will be in the form `<namespacePrefix>.[<druid service name>].[<druid h
 User has control of `<namespacePrefix>.[<druid service name>].[<druid hostname>].`
 
 You can omit the hostname by setting `ignoreHostname=true`
-`druid.SERVICE_NAME.dataSourceName.queryType.query.time`
+`druid.SERVICE_NAME.dataSourceName.queryType.query/time`
 
 You can omit the service name by setting `ignoreServiceName=true`
-`druid.HOSTNAME.dataSourceName.queryType.query.time`
+`druid.HOSTNAME.dataSourceName.queryType.query/time`
+
+Elements in metric name by default are separated by "/", so graphite will create all metrics on one level. If you want to have metrics in the tree structure, you have to set `replaceStashWithDot=true`
```

Comment:
`replaceSlashWithDot` is configuration option for converters, same as `ignoreHostname` and `ignoreServiceName`.